tests/unittests/OperatorGradTest.cpp

```diff
@@ -105,6 +105,57 @@ TEST_P(OperatorGradTest, concat) {
   EXPECT_TRUE(expectedRight.isEqual(*getGradTensor(varGrads, B)));
 }
 
+TEST_P(OperatorGradTest, conv) {
+  ENABLED_BACKENDS(Interpreter, CPU);
+
+  auto *X = mod_.createPlaceholder(
+      ElemKind::FloatTy, {1, 3 /* width */, 3 /* height */, 1 /* depth */}, "x",
+      false);
+  bindings_.allocate(X)->getHandle() = {0, 1, 2, 3, 4, 5, 6, 7, 8};
+
+  ConvolutionNode *O =
+      F_->createConv(bindings_, "conv", X, 1 /* outChannels */, 2 /* kernel */,
+                     1 /* stride */, 0 /* pad */, 1 /* group */);
+
+  // Note that filter is trainable, but probably it's okay since we run the
+  // graph only once.
+  bindings_.get(llvm::cast<Placeholder>(O->getFilter()))->getHandle() = {0, 1,
+                                                                         2, 3};
+  bindings_.get(llvm::cast<Placeholder>(O->getBias()))->getHandle() = {1};
+
+  VariableGradientsList varGrads = computeVarGrads(O);
+
+  // (Note that T stands for the filter).
+  // O_0_0 = X_0_0*T_0_0 + X_0_1*T_0_1 + X_1_0*T_1_0 + X_1_1*T_1_1 + b = 20
```

Comment:
Consider writing these equations as

```
O_{0,0} = X_{0,0}*T_{0,0} + X_{0,1}*T_{0,1} + X_{1,0}*T_{1,0} + X_{1,1}*T_{1,1} + b = 20
```

I think they will be clearer this way.